cmd/api/main.go

```diff
@@ -47,10 +47,12 @@ func main() {
 	if user == "" || pass == "" {
 		log.Fatal("Must provide both DB_USER and DB_PASSWORD")
 	}
-	// Connect to the MySQL database.
-	// DSN derived from https://github.com/go-sql-driver/mysql#dsn-data-source-name
-	dbURI := fmt.Sprintf("%s:%s@%s(%s)/%s?parseTime=true", user, pass, os.Getenv("DB_PROTOCOL"), os.Getenv("DB_ADDR"), os.Getenv("DB_NAME"))
-	db, err := gorm.Open(mysql.Open(dbURI), &gorm.Config{})
+	// Connect to the database.
+	// DSN derived from https://pkg.go.dev/gorm.io/driver/postgres
+
+	dbURI := fmt.Sprintf("host=%s user=%s password=%s dbname=%s port=5432", os.Getenv("DB_ADDR"), user, pass, os.Getenv("DB_NAME"))
+	//dbURI := fmt.Sprintf("%s:%s@%s(%s)/%s?parseTime=true", user, pass, os.Getenv("DB_PROTOCOL"), os.Getenv("DB_ADDR"), os.Getenv("DB_NAME"))
```

Comment:
suggest removing or leaving a comment explaining the alternate usage

Reply:
Removed (this was the old mysql format).